### PR TITLE
Update routing_content_core_6.partial.md

### PR DESCRIPTION
**Pull request by seanfarmar**

### CODE REVIEW

File: transports/msmq/routing_content_msmqtransport_[1,).partial.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Mapping the logical destination to the physical address containing the queue and host names is the responsibility of physical routing. 
 
+
```
Comment (andreasohlund) [inline transports/msmq/routing_content_msmqtransport_[1,).partial.md]: Notpick but if the note is the same in all 3 places we should pull it into an "include"

https://github.com/Particular/docs.particular.net#markdown-includes

File: transports/msmq/routing_content_core_[,6).partial.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
+
+NOTE: When using instance mapping, the settings will have no effect on **audit/error** queue and **publish/subscribe** (except for subscription messages, subscription messages will be sent to all listed instances)
```
Comment (timbussmann) [inline transports/msmq/routing_content_core_[,6).partial.md]: I think there is no instance mapping file in NSB < v6, so this note wouldn't make much sense here, does it?

File: transports/msmq/routing_content_core_6.partial.md [hunk 3]
```diff
@@ -3,6 +3,9 @@ Mapping the logical destination to the physical address containing the queue and
 The preferred way of configuring the physical routing is via the instance mapping file.
 
 
+NOTE: When using instance mapping, the settings will have no effect on **audit/error** queue and **publish/subscribe** (except for subscription messages, subscription messages will be sent to all listed instances)
```
Comment (timbussmann) [inline transports/msmq/routing_content_core_6.partial.md]: shouldn't this be within the instance mapping file section?
should it be spelled out "audit and error queues"?
do we need to mention pubsub?

Comment (SzymonPobiega) [reply transports/msmq/routing_content_core_6.partial.md]: I get why we mention it does not include audit/error but why mentioning pub/sub here?

Comment (andreasohlund) [reply transports/msmq/routing_content_core_6.partial.md]: Doesn't it make sense to clarify that when publishing to only the instances that have subscribed will get the event regardless what's present in the mapping file?

Comment (timbussmann) [reply transports/msmq/routing_content_core_6.partial.md]: That way of wording makes more sense than stating that pubsub is t affected except for subscription messages.

File: transports/msmq/routing_content_core_[,6).partial.md (hunk 2 above)
Comment (seanfarmar) [reply transports/msmq/routing_content_core_[,6).partial.md]: Good point, so keep the partial and have it for 6 and up?

**Comment by seanfarmar** [comment]
I've removed it from `routing_content_core_[,6).partial.md` and moved the note into the 
`Instance mapping file` section.

And changed the wording... Hope this is better...

What do you think?

File: transports/msmq/routing_content_msmqtransport_[1,).partial.md [hunk 4]
```diff
@@ -5,6 +5,11 @@ The preferred way of configuring the physical routing is via the instance mappin
 
 ## Instance mapping file
 
+NOTE: When using instance mapping: 
+* The settings will have no effect on **audit and error queues**.
+* **publish/subscribe**: The publisher will be publishing messages only to the endpoint instances that have subscribed to the events, ignoring the settings in the mapping file (the adress of the subscriber that was provided in the subscription messgae will be used).
```
Comment (andreasohlund) [inline transports/msmq/routing_content_msmqtransport_[1,).partial.md]: **publish/subscribe** => **publishing**

File: transports/msmq/routing_content_msmqtransport_[1,).partial.md [hunk 5]
```diff
@@ -5,6 +5,11 @@ The preferred way of configuring the physical routing is via the instance mappin
...
+* **publish/subscribe**: The publisher's instances should be entered in the Instance mapping file (so the subscribers can send a subscription message to them)
```
Comment (andreasohlund) [inline transports/msmq/routing_content_msmqtransport_[1,).partial.md]: **publish/subscribe** => subscriptions
```suggestion
* **subscribing**: Subscription messages will be sent to all publisher instances listed in the Instance mapping file
```

**Comment by seanfarmar** [comment]
Updated

File: transports/msmq/routing_content_core_6.partial.md [hunk 6]
```diff
@@ -2,9 +2,13 @@ Mapping the logical destination to the physical address containing the queue and
 
 The preferred way of configuring the physical routing is via the instance mapping file.
 
-
 ## Instance mapping file
 
+NOTE: When using instance mapping: 
+* The settings will have no effect on **audit and error queues**.
```
Comment (timbussmann) [inline transports/msmq/routing_content_core_6.partial.md]: does the Note-section allow for bullet points?

Comment (seanfarmar) [reply transports/msmq/routing_content_core_6.partial.md]: I'm not sure...

**Comment by mauroservienti** [comment]
What’s missing here to get this in?

**Comment by mauroservienti** [comment]
@seanfarmar ping

**Comment by seanfarmar** [comment]
@mauroservienti I think it's all done if we have reviewers happy?

File: transports/msmq/routing_content_core_6.partial.md (hunk 6 above)
Comment (DavidBoike) [reply transports/msmq/routing_content_core_6.partial.md]: @timbussmann If you do it like this https://github.com/Particular/docs.particular.net#multi-line

Comment (mauroservienti) [reply transports/msmq/routing_content_core_6.partial.md]: Updated to make the note multi-line.

Comment (timbussmann) [reply transports/msmq/routing_content_core_6.partial.md]: thanks @DavidBoike and @mauroservienti ❤️

**Pull request merged**